.ci/post_review.py

```diff
@@ -0,0 +1,214 @@
+#! usr/bin/env python3
+
+# Automatically create review from clang-tidy report
+# Copyright (c) Ashar Khan <ashar786khan.at.gmail.com>
```

Comment:
Suggestion to put the date of publication.
```py
# Script for automatically creating review on github from clang-tidy report
# Copyright (c) Ashar Khan <ashar786khan.at.gmail.com> 2020
```

Reply:
```suggestion
# Copyright (c) 2020 Ashar Khan <ashar786khan.at.gmail.com>
```